app/src/main/java/space/taran/arknavigator/mvp/model/repo/meta/PlainMetadataStorage.kt

```diff
@@ -34,24 +23,25 @@ class PlainMetadataStorage(val root: Path) : MetadataStorage {
         metaDir.createDirectories()
     }
 
-    override fun locate(path: Path, resource: ResourceMeta): Map<MetaExtraTag, String?>? {
+    override fun locate(path: Path, resource: ResourceMeta): ResourceMeta? {
         val metadata = metaPath(resource.id)
         if (!Files.exists(metadata)) {
             Log.w(METADATA, "metadata was not found for resource $resource")
             // means that we couldn't generate anything for this kind of resource
```

Comment:
Probably we should treat this case as that we don't have extra metadata (that is "metadata" for a user), but still return `resource: ResourceMeta` parameter as a result, since we can't have a resource without metadata (from systemic view). Lack of `kind` would mean the resource is of unknown type. That would convert return type from `ResourceMeta?` to `ResourceMeta`.

@mdrlzy What do you think?